zhaquirks/ikea/twobtnremote.py

```diff
@@ -85,7 +85,7 @@ class IkeaTradfriRemote2Btn(CustomDevice):
                 DEVICE_TYPE: zha.DeviceType.NON_COLOR_CONTROLLER,
                 INPUT_CLUSTERS: [
                     Basic.cluster_id,
-                    DoublingPowerConfig1CRCluster,
+                    PowerConfiguration.cluster_id,
```

Comment:
Don't seems related to your quirk. It is needed?

Reply:
Yeah, let's not change the old two button dimmer in this PR.

The problem is that newer IKEA firmware versions didn't change the device signature for the (old) two button dimmer.
We shouldn't double battery percentage on those newer firmware versions, but still on old firmware. But we can't match by firmware version at the moment.

The signature also isn't "refreshed" when just updating the firmware through ZHA. I think it needs to be removed from ZHA and completely re-added. (I don't think re-pairing is enough, but I'm not sure right now)

---

Without matching per firmware version, maybe it's possible to find out what voltage 0% and 100% respond to on IKEA remotes. @MattWestb Any idea? 😄 
At least for the two button dimmer, we could use a linear function to calculate the battery percentage based on the battery voltage (like we do for some other devices). That way it would work regardless of the firmware version -- assuming IKEA sends a proper battery voltage.

Or maybe we just do it the Z2M way and remove doubling everywhere -- regardless of firmware version.

Reply:
My "beloved" d(f)conz have just removing it for all IKEA controllers then they is expecting all user updating the firmware and its one way to going.
But i think its better doing one separate PR for that and doing all but not the old rotating dimmer then its still on 2.3.X firmware and the classes for older ZLL and also ZB3 versions. 

If i remember right is most reporting OK battery voltage but first gen Symfonisk is no (i think 25V is full) and perhaps some more but its not easy testing that all my controllers is on latest firmware without j-tag flashing all types them with one older firmware and testing.

Z2M is doing matching on SW_Build for gen 3 controller and i think on the old ones 2.

ZHA is having the SW_build and version saved in the Zigbe.DB so one way is implanting quirk to using it and its all good but is it needed for most users ?

Reply:
Reverted that unrelated change in https://github.com/zigpy/zha-device-handlers/pull/2609/commits/cbfb4d50a78da0fe04f5c99580be6c2e7eed0f22

We still need to address this, but preferably without breaking outdated or updated dimmers.
Since the battery voltage should be reported the same (regardless of if it's updated or not), we should be able to map voltage to percentage manually and just ignore the percentage the device sends.
We really need to find out how the device maps battery voltage to percentage though.
Would be amazing if somebody can give us multiple data points on which voltage corresponds to which percentage.

Reply:
Then you need verifying that all firmware versions is sending correct voltage then i have see dome IKEA controllers sending no or wrong voltage (Symfonisk first gen was sending 25V and not 2.5V in some version).
I think better taking the double for the quirk class that also being loaded by 24.x firmware if have the same as the classes and letting all old have the doubling and user shall updating there devices for fixing battery darning and network loss then running battery low.